backend/src/services/calendar/core/serializers.py

```
@@ -0,0 +1,1009 @@
+    
+    return result
+
+
+# ============================================================================
+# CALENDAR LIST ENTRY SERIALIZER
+# ============================================================================
+
+
+def serialize_calendar_list_entry(
+    entry: CalendarListEntry,
+    user_email: Optional[str] = None,
```

Ruff (0.14.14)

[warning] Unused function argument: `user_email`

(ARG001)